pkg/cmd/get/get_build_logs.go

```diff
@@ -233,7 +233,8 @@ func (o *GetBuildLogsOptions) getLastJenkinsBuild(name string, buildNumber int)
 	}
 }
 
-func (o *GetBuildLogsOptions) getProwBuildLog(kubeClient kubernetes.Interface, tektonClient tektonclient.Interface, jxClient versioned.Interface, ns string, tektonEnabled bool) error {
+// GetProwBuildLog prompts the user, if needed, to choose a pipeline, and then prints out that pipeline's logs.
+func (o *GetBuildLogsOptions) GetProwBuildLog(kubeClient kubernetes.Interface, tektonClient tektonclient.Interface, jxClient versioned.Interface, ns string, tektonEnabled bool) error {
```

Comment:
So how does this change pan out in the end? `jx get build log` lists meta pipeline build as different items for which the logs can be retrieved?

Reply:
How can I see how this looks like in the end?

Reply:
`jx get build log` will treat the meta pipeline and execution pipeline as a single build, with the logs for all steps in both of them included in the proper order.